### PR TITLE
Issue 1166 council testcase

### LINKED ISSUE #1166: Test case: Council test case

### PR DESCRIPTION
**Pull request by Ekaanth**

### CODE REVIEW

File: mocks/handlers/councilHandler.js [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+          {
+            __typename: 'SubstrateChainCouncilMember',
+            account: {
+              __typename: 'SubstrateChainAccount',
+              address: '1hCMdtRsaRA4ZTEKpPKPvEjK9rZpGhyFnRHSDhqFMCEayRL',
+              display: 'RTTI-5220 (POLKADOT)',
+            },
+            backing: '217343856918926861',
+            formattedBacking: '21.7343  MDOT',
+            voters: [
```
Comment (ishrak-imam) [inline mocks/handlers/councilHandler.js]: Let's reduce the items in the voters list to 2 or 3. We mainly care about the shape of the data rather the exact data.

File: src/testUtils.tsx [hunk 2]
```diff
@@ -13,9 +14,9 @@ export const client = new ApolloClient({
 
 function Providers({children}: {children: React.ReactNode}) {
   return (
-    <ThemeProvider>
-      <ApolloProvider client={client}>{children}</ApolloProvider>
-    </ThemeProvider>
+    <LitentryApiClientProvider>
```
Comment (ishrak-imam) [inline src/testUtils.tsx:18]: Can you please create a ticket to clean this up later. In the LitentryApiClientProvider we have an instance of Apollo client already. And we are creating a new Apollo client here also, which is needed in the `setupTests.js` file. Ideally we would use the same instance of the Apollo client in test environment too.

For now this implementation unblocks our tests, but please include the ticket here with a TODO.

File: src/ui/screens/Council/CouncilScreen.tsx [hunk 3]
```diff
@@ -352,10 +360,10 @@ function SubmitCandidacyModel({visible, setVisible, moduleElection}: SubmitCandi
       ) : null}
       <Padder scale={1} />
       <View style={styles.buttons}>
-        <Button onPress={reset} mode="outlined" compact>
+        <Button onPress={reset} mode="outlined" compact testID="submit-cancel-button">
```
Comment (ishrak-imam) [inline src/ui/screens/Council/CouncilScreen.tsx]: NIT:
```suggestion
        <Button onPress={reset} mode="outlined" compact testID="submit-candidacy-cancel-button">
```

File: src/ui/screens/Council/CouncilScreen.tsx [hunk 4]
```diff
@@ -352,10 +360,10 @@ function SubmitCandidacyModel({visible, setVisible, moduleElection}: SubmitCandi
...
           Cancel
         </Button>
-        <Button mode="contained" disabled={submitCandidacy} onPress={onSubmitCandidacy}>
+        <Button mode="contained" disabled={submitCandidacy} onPress={onSubmitCandidacy} testID={'submit-button'}>
```
Comment (ishrak-imam) [inline src/ui/screens/Council/CouncilScreen.tsx]: NIT:
```suggestion
        <Button mode="contained" disabled={submitCandidacy} onPress={onSubmitCandidacy} testID={'submit-candidacy-button'}>
```

**Review by ishrak-imam — APPROVED** [review]

File: src/testUtils.tsx (hunk 2 above)
Comment (Ekaanth) [reply src/testUtils.tsx:18]: created an issue here: https://github.com/litentry/litentry-app/issues/1275

**Pull request merged**